Tools/WebKitTestRunner/ios/UIScriptControllerIOS.h

```diff
@@ -121,6 +121,7 @@ class UIScriptControllerIOS final : public UIScriptControllerCocoa {
     JSObjectRef inputViewBounds() const override;
     JSRetainPtr<JSStringRef> scrollingTreeAsText() const override;
     JSRetainPtr<JSStringRef> uiViewTreeAsText() const override;
+    JSRetainPtr<JSStringRef> caLayerTreeAsText() const override;
```

Comment:
I think this should go in UIScriptControllerCocoa -- CA is identical on both platforms.